docs/_posts/2020-08-20-42.2.16-release.md

```diff
@@ -0,0 +1,61 @@
+---
+title: PostgreSQL JDBC Driver 42.2.16 Released
+date: 2020-08-20 15:10:04 +0000
+categories:
+  - new_release
+version: 42.2.16
+---
+**Notable changes**
+
+### Changed
+
+### Added
+
+### Fixed
+
+## [42.2.16] (2020-08-20)
+
+### Fixed
+- Arrays sent in binary format are now sent as 1 based. Fixes [issue 1860](https://github.com/pgjdbc/pgjdbc/issues/1860) in [PR 1863](https://github.com/pgjdbc/pgjdbc/pull/1863).
```

Comment:
I guess we should add something like `regression since 42.2.15`

Reply:
I think it has to be a bit more nuanced than that. It is a regression in 42.2.15 for multi-dimensional arrays, varchar/text, bytea and oid. For single dimensional primitive types (short, int, long, float, double, boolean), the 0-based behavior has been present since their original support was added in 42.2.0.

Reply:
OK, I will change that, thanks for the review

Reply:
Do you consider an array of text to be multi-dimensional? It was a regression for that which flagged the problem

Reply:
The wording on this is a bit tricky. In 42.2.15, any array type which could binary encoded as a single dimensional array could now also be binary encoded for multi-dimensional arrays. So in 42.2.14, a long[] would be binary encoded, but a long[][] would not be. Starting in 42.2.14, the long[][] could also be binary encoded. In 42.2.15, I also introduced the ability to binary encode varchar/text, bytea[], oid, and the Object forms of the primitive types supported in 42.2.0. So these array types regressed in both single dimension and multi-dimensional in 42.2.15.

Reply:
Just an idea: what if we polish the release notes after the release? 😉 
The impact is that Dependabot might have already started to update pgjdbc all over the place 🙀 , and we might want to release .16 sooner than later 🚀